src/Symfony/Bridge/PhpUnit/Legacy/CoverageListenerTrait.php

```diff
@@ -78,13 +76,45 @@ public function startTest($test)
             }
         }
 
+        if (class_exists(Registry::class)) {
+            $this->addCoversForDocBlockInsideRegistry($test, $covers);
+
+            return;
+        }
+
+        $this->addCoversForClassToAnnotationCache($test, $covers);
+    }
+
+    private function addCoversForClassToAnnotationCache($test, $covers)
+    {
+        $testClass = \PHPUnit\Util\Test::class;
```

Comment:
```suggestion
        $testClass = Test::class;
```

Reply:
Thank you. I removed everything I happen [to inherit from 3.4](https://github.com/symfony/symfony/blob/4351a70637105102f832216e4b6f6cbe1cb9ab0f/src/Symfony/Bridge/PhpUnit/Legacy/CoverageListenerTrait.php#L68-L71).